core/src/reporter/utils.ts

```diff
@@ -252,3 +255,38 @@ export async function sendTransactionCaver({
     throw new OraklError(OraklErrorCode.CaverTxTransactionFailed)
   }
 }
+
+export function isSubmitMethod(wallet: CaverWallet, payload: string) {
```

Comment:
is there a specific reason for wallet parameter to be `CaverWallet` type? it could be nicer to use `ethers.js` libarary for general usage in evm based chains.

Reply:
reference:
https://github.com/ethers-io/ethers.js/issues/478#issuecomment-495814010